internal/debug/profiler.go

```diff
@@ -0,0 +1,65 @@
+package debug
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
+	"gopkg.in/urfave/cli.v1"
+)
+
+var (
+	datadogProfilerFlag = cli.BoolFlag{
+		Name:  "datadog.profiler",
+		Usage: "Enable Datadog profiler",
+	}
+	datadogProfilerExpensiveFlag = cli.BoolFlag{
+		Name:  "datadog.profiler.expensive",
+		Usage: "Enable Datadog profiler expensive metrics",
+	}
+	datadogProfilerEnvironmentFlag = cli.StringFlag{
+		Name:  "datadog.profiler.environment",
+		Usage: "Set Datadog profiler environment",
+		Value: "",
+	}
+	datadogProfilerTagsFlag = cli.StringFlag{
+		Name:  "datadog.profiler.tags",
+		Usage: "Comma-separated tags (key:value) attached to all measurements",
+		Value: "",
+	}
+)
+
+func init() {
+	Flags = append(Flags,
```

Comment:
I don't like this @ferranbt, can we just add these to the `var Flags` declaration even if it's changing geth upsteam?